### PR TITLE
[release/v2.17] Bump machine controller to v1.27.8

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: bump machine controller to v1.27.6 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

```release-note
bump machine controller to v1.27.8
```

### CODE REVIEW

**Review by kron4eg — APPROVED** [review]
/approve

**Comment by zreigz** [comment]
@kron4eg I had to call `go mod tidy` Need LGTM again :)

**Comment by kron4eg** [comment]
/hold

@zreigz something is wrong here

**Comment by zreigz** [comment]
@kron4eg every test fails because: `  digitalocean_test.go:98: cluster nodes not ready: NodeDeployment has not reached 1 ready replicas` Control plane is always ready but can't create machine nodes

**Comment by zreigz** [comment]
@kron4eg 
I run this environment locally

err

**Comment by zreigz** [comment]
From machine controller:
```
E0614 07:41:47.533581       1 node_csr_approver.go:77] Reconciliation of request /csr-j24c2 failed: failed to approve CSR "csr-j24c2": CertificateSigningRequest.certificates.k8s.io "csr-j24c2" is invalid: status.conditions[0].status: Required value

```

**Comment by kron4eg** [comment]
@zreigz this is super strange, the only diff is related to openstack/vSphere custom CA

https://github.com/kubermatic/machine-controller/compare/v1.27.5...v1.27.6

**Comment by zreigz** [comment]
I used MC v1.30 in my local environment and have the same problem

**Comment by kron4eg** [comment]
I've recently updated APIs, including certificates but those changes are not yet released and MC v1.30 doesn't contain them o_O

File: pkg/resources/machinecontroller/deployment.go
Comment (kron4eg) [inline pkg/resources/machinecontroller/deployment.go]: Please use `v1.27.8`

**Review by kron4eg — APPROVED** [review]
/approve
/retest

**Pull request merged**
